### DISCUSSION TITLE
Mouse support?

### DISCUSSION THREAD

**Discussion by NEONZARIN** [cataclysmbn/Cataclysm-BN#2844] [General]
Hello. Can you add this?
https://github.com/CleverRaven/Cataclysm-DDA/pull/40794
https://github.com/CleverRaven/Cataclysm-DDA/pull/41272

**Comment by Coolthulhu** [comment]
I'll check once it's done.

**Comment by VuradoZ** [comment]
And this https://github.com/CleverRaven/Cataclysm-DDA/pull/41605

**Comment by Coolthulhu** [comment]
User request on Discord: Also make Y/N prompts clickable.

**Comment by leemuar** [comment]
ok, I've finally figured out what introduced mouse movement bug after https://github.com/cataclysmbn/Cataclysm-BN/pull/6195

Turned out the CDDA source code had theat bug too after I've made the same thing there. It was just hackfixed there by other dev. So there are two ways to fix that - make the same small hackfix or make a better change (but it requires to rewrite more code)
